### PR TITLE
Add post-ice save tool

### PR DESCRIPTION
**Pull request by thisisnotyoho**

### CODE REVIEW

**Comment by FoxLisk** [comment]
i should probably test that the save files this generates actually work but using an emulator is cringe so ill do it tomorrow

File: save_files/lib.js [hunk 1]
```diff
@@ -192,30 +203,44 @@ function bottle_updates(tavern, vendor) {
...
+        return [[ADDRESSES['mushroom'], 2]];
+    } else {
+        return [[ADDRESSES['mushroom'], 0]];
+    }
+}
 
 function form_to_updates(form) {
     let fields = [
         ['arrows', new InputToStuff(
-            numeric, a => a >= 0 && a <= 15, address_to_value('arrows')
```
Comment (FoxLisk) [inline save_files/lib.js:199]: oh i messed this up, will need to either de-genericize or find a better way to do so

can probably set min/max attrs on the inputs themselves and crib off of those here

File: save_files/lib.js [hunk 2]
```diff
@@ -192,30 +203,44 @@ function bottle_updates(tavern, vendor) {
...
         )],
         ['rupees', new InputToStuff(
-            numeric,  a => a >= 0 && a <= 150, rupee_updates,
-        )],
-        ['mushroom', new InputToStuff(
-            checked, () => true, address_to_value('mushroom')
+            numeric,  a => a >= 0 && a <= 250, rupee_updates,
         )],
         ['sanc_heart', new InputToStuff(
-            checked, () => true, sanc_heart_updates,
```
Comment (FoxLisk) [inline save_files/lib.js:211]: oh i messed this up too! this info needs to live on the sanc heart select

**Comment by thisisnotyoho** [comment]
looked over and tested the changes, everything looks to be working

**Pull request merged**